tests/roots/test-ext-autodoc/target/inherited_annotations.py

```diff
@@ -0,0 +1,16 @@
+"""
+    Test case for #11387 corner case involving inherited
+    members with type annotations on python 3.9 and earlier
+"""
+
+class HasTypeAnnotatedMember:
+    inherit_me: int
+    """Inherited"""
+
+class NoTypeAnnotation(HasTypeAnnotatedMember):
+    a = 1
+    """Local"""
+
+class NoTypeAnnotation2(HasTypeAnnotatedMember):
+    a = 1
+    """Local"""
```

Comment:
I think this file is missing one LF at the end (at least it appears so on github but I don't remember whether the other testroots files have one or not actually).

Reply:
fixed